src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/UserDefinedEditorBehaviorFeature.java

```diff
@@ -62,4 +62,11 @@ public boolean isEnableTextMateNestedBracesImprovements(@NotNull PsiFile file) {
         // Note that this defaults to enabled for user-defined language server definitions
         return (editorSettings == null) || editorSettings.enableTextMateNestedBracesImprovements;
     }
+
+    @Override
+    public boolean isEnableTextMateFileViewProvider(@NotNull PsiFile file) {
```

Comment:
Feature flag defaults to `true`/enabled for user-defined language server definitions.